### PR TITLE
ui: add merge queue graph to queues dashboard

### PR DESCRIPTION
**Pull request by benesch**
Release note: None

### CODE REVIEW

**Comment by benesch** [comment]
image

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/queues.tsx [hunk 1]
```diff
@@ -68,6 +68,13 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
+    <LineGraph title="Merge Queue" sources={storeSources}>
+    <Axis units={AxisUnits.Count} label="actions">
+      <Metric name="cr.store.queue.merge.process.success" title="Successful Actions / sec" nonNegativeRate />
+      <Metric name="cr.store.queue.merge.pending" title="Pending Actions" downsampleMax />
+    </Axis>
+    </LineGraph>,
```
Comment (vilterp) [inline pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/queues.tsx]: lint: indent `Axis`:

```js
    <LineGraph title="Merge Queue" sources={storeSources}>
      <Axis units={AxisUnits.Count} label="actions">
        <Metric name="cr.store.queue.merge.process.success" title="Successful Actions / sec" nonNegativeRate />
        <Metric name="cr.store.queue.merge.pending" title="Pending Actions" downsampleMax />
      </Axis>
    </LineGraph>,
```

Comment (benesch) [reply pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/queues.tsx]: Oops, thanks. Done.

**Review by vilterp — APPROVED** [review]

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/queues.tsx (hunk 1 above)
Comment (vilterp) [reply pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/queues.tsx]: Np. I should find the linter setting that catches stuff like this; seems like it's not on…

**Comment by benesch** [comment]
Test failure is just a flake. Merging! Thanks for the fast review!

bors r=vilterp

**Pull request merged**